powerstore/datasource_volume_group_test.go

```diff
@@ -0,0 +1,58 @@
+package powerstore
+
```

Comment:
Please add more tests like providing both ID and name, invalid ID

Reply:
done